utils.sh

```diff
@@ -30,6 +30,13 @@ function create_cluster() {
     $GOPATH/src/github.com/openshift-metalkube/kni-installer/bin/kni-install --dir "${assets_dir}" --log-level=debug create cluster
 }
 
+function wait_for_cvo_finish() {
+    local assets_dir
+
+    assets_dir="$1"
+    $GOPATH/src/github.com/openshift-metalkube/kni-installer/bin/kni-install --dir "${assets_dir}" --log-level=debug upi finish
```

Comment:
Doesn't this require that all operators come up? Are we there yet? I thought the last successful run I had not everything came up

Reply:
Yes, it would. In my tests the only misbehaving operator was image-registry - and #262 would fix that